hot-reload-devtools/src/main/kotlin/org/jetbrains/compose/devtools/sidecar/DtAnimatedWindow.kt

```diff
@@ -208,25 +135,19 @@ internal fun animateWindowPosition(
     }
 }
 
-internal fun DpSize.toDimension(): Dimension = Dimension(width.value.toInt(), height.value.toInt())
+internal val attachedSidecarWindowSize: DpSize = DpSize(
+    width = DtSizes.largeLogoSize + DtPadding.small * 2,
+    height = DtSizes.iconSize * 9 + DtPadding.small * 14,
+)
```

Comment:
I think hardcoding the size here would work (how often do we really add elements to this toolbar, right?), however, its certainly not super elegant. 

AFAIK there is a way (with new versions of compose) to auto-size the window to the contents. This seems really nice!

Reply:
I currently only know of the way to auto-size the window to fit its contents on creation by setting its size to `Unspecified`. But our window changes its height dynamically, so this solution does not work.

If there are ways to dynamically adjust the window size --- that would be really cool! I will look into it

Reply:
Then lets just contiue like originally proposed by you!